code-challenges.js

```diff
@@ -9,38 +9,111 @@
 // --------------------1) Create a function that takes in a number and determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+// I want to create a function expression that will take in the temp variables.
+//I should use some if/else statements to determine if the temp is boiling or below boiling
+//I want to return a string
+//potentially check if a given input is valid 
+
+
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
+const temperatureChecker = (temp) => {
+   if(typeof temp !== "number") return "Please enter a valid number";
+   
+   if(temp === 212){
+      return `${temp} is boiling point`; 
+   } else if(temp < 212) {
+     return `${temp} is below boiling point`;  
+   } else {
+    return `${temp} is above boiling point`;    
+   }
+}
+
+//console.log(temperatureChecker(temp3));
```

Comment:
Looks good! A refactor could be to reduce the number of individual if statements by having the initial check be a part of your temperature check.